lib/open_api/serializer.rb

```diff
@@ -0,0 +1,14 @@
+module OpenApi
+  module Serializer
+    def as_json(arg = {})
+      return super unless self.kind_of?(ApplicationRecord)
```

Comment:
@gmcculloug I believe we'll need a list of classes here to check for.  There are a number of services that will require the `as_json` method and they will will skip the `unless` conditional I believe.